crates/uv-cli/src/lib.rs

```diff
@@ -2606,30 +2606,83 @@ pub struct ToolNamespace {
 
 #[derive(Subcommand)]
 pub enum ToolCommand {
-    /// Run a tool.
+    /// Run a command provided by a Python package.
+    ///
+    /// By default, the package to install is assumed to match the command name.
+    ///
+    /// The name of the command can include an exact version in the format
+    /// `<package>@<version>`, e.g., `uv run ruff@0.3.0`. If more complex
+    /// version specification is desired or if the command is provided by a
+    /// different package, use `--from`.
+    ///
+    /// If the tool was previously installed, i.e., via `uv tool install`, the
+    /// installed version will be used unless a version is requested or the
+    /// `--isolated` flag is used.
+    ///
+    /// A `uvx` alias is provided for `uv tool run` for quick invocations.
```

Comment:
I might say... uvx is provided as a convenient alias for uv tool run.

Reply:
```suggestion
    /// `uvx` is provided as a convenient alias for `uv tool run`, their behavior is identical.
```